### DISCUSSION TITLE
Auto-close comments

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#357] [Feature Requests]
> Can this plugin auto close or disable comments on a particular post (or all posts) , at a specific time? (Just as it can schedule time for post to expire)
> 
> Instead of expiring, could it just close/hide the comment box on that particular post?
> 
> E.g to close/hide comment box on a post by 12pm (so users can still see the post but can’t comment after 12pm)
> 
> Can this plugin do this??

https://wordpress.org/support/topic/auto-disable-expire-or-close-comments-at-specific-time/

**Comment by andergmartins** [comment]
We've implemented this for workflows by using the step "Update post".
